src/13_file_io.py

```diff
@@ -10,10 +10,15 @@
 # Note: pay close attention to your current directory when trying to open "foo.txt"
 
 # YOUR CODE HERE
+with open("/Users/ramon/Documents/lambda/06.ComputerScience/01.IntroToPythonAndOPP/Intro-Python-I/src/foo.txt") as foo:
+    print(foo.read())
 
 # Open up a file called "bar.txt" (which doesn't exist yet) for
 # writing. Write three lines of arbitrary content to that file,
 # then close the file. Open up "bar.txt" and inspect it to make
 # sure that it contains what you expect it to contain
 
-# YOUR CODE HERE
\ No newline at end of file
+# YOUR CODE HERE
+with open("/Users/ramon/Documents/lambda/06.ComputerScience/01.IntroToPythonAndOPP/Intro-Python-I/src/bar.txt", "w") as bar:
```

Comment:
be mindful that code you write needs to be used by other users. keep in mind when people review your code they are likely going to be pulling the code down to their local machine.